pkg/venafi/tpp/connector.go

```diff
@@ -403,6 +403,33 @@ func prepareRequest(req *certificate.Request, zone string) (tppReq certificateRe
 	tppReq.CASpecificAttributes = append(tppReq.CASpecificAttributes, nameValuePair{Name: "Origin", Value: origin})
 	tppReq.Origin = origin
 
+	if req.ValidityHours > 0 {
+
+		issuerHint := ""
```

Comment:
I think it would be clearer to call this the `expirationDateAttribute` since it is being used to specify the Config attribute we're using to specify the desired expiration date of the certificate being requested.

Reply:
fixed on most recent commit, changed ValidityHours to expirationDateAttribute

Reply:
@angelmoo I think you misunderstood me.  I was asking that the `issuerHint` local variable here be renamed to `expirationDateAttribute` because it's values are things like "Specific End Date" and "Microsoft CA:Specific End Date" which are TPP Config attribute names. `ValidityHours` was an appropriate name for the `Request` struct as I noted where it is declared in certificate.go.  Apologies for the confusion.

Reply:
fixed on most recent commit, reverted back the changes and changes issuerHint to expirationDateAttribute,

sorry for the misunderstanding